cobbler/modules/authentication/spacewalk.py

```diff
@@ -113,6 +114,9 @@ def authenticate(api_handle: "CobblerAPI", username: str, password: str) -> bool
     if api_handle is None:
         raise CX("api_handle required. Please don't call this without it.")
     server = api_handle.settings().redhat_management_server
+    if api_handle.settings().spacewalk_authentication_endpoint:
```

Comment:
The new variable should define the server and method. Fall back (if not defined) should be the exact same behaviour as before, to make sure we don't break any environment.